sound/Music.java

```diff
@@ -50,9 +50,10 @@ public Music(int instr) {
     /**
      * Construct Music with specified channels and instrument.
      * @param numCh the number of channels
-     * @param instr the integer code for the midi instrument to use for all channels
+     * @param instr the integer code for the midi instrument
+     * to use for all channels
```

Comment:
It seems like the second line of this comment should be indented for readability.